src/concrete/ml/torch/hybrid_model.py

```diff
@@ -1,16 +1,44 @@
 """Implement the conversion of a torch model to a hybrid fhe/torch inference."""
 
+import ast
+import enum
+import io
+import sys
+import time
 import uuid
 from pathlib import Path
-from typing import List, Optional, Union
+from typing import Dict, List, Optional, Tuple, Union
 
+import numpy
+import requests
 import torch
 from concrete.fhe import Configuration
 from torch import nn
 from transformers import Conv1D
 
 from ..deployment.fhe_client_server import FHEModelClient, FHEModelDev
-from .compile import compile_torch_model
+from .compile import QuantizedModule, compile_torch_model
```

Comment:
I realize we have a problem here. It won't work for QAT models from brevitas.

Reply:
what do you mean ?

Reply:
The hybrid model uses compile_torch_model only so it won't work for brevitas networks.

Reply:
ah indeed, but maybe there's not much more to do for supporting that feature right ?

Reply:
Yes should be straightforward. We can do this in another PR if it requires another CI run and more tests.

Reply:
Alright with you if we tackle this in another PR?

Reply:
Yes sure. Just make a TODO + issue please.

Reply:
Yes ofc